prefq/server.py

```diff
@@ -55,14 +61,53 @@
 DEFAULT_DEBUG = False
 
 
-def before_first_request():
+def before_first_request(sshkey):
     """Define starting routine"""
 
+    if sshkey is not None:
+        with open(sshkey, "rb") as key_file:
+            app.config["PUBLIC_KEY"] = serialization.load_ssh_public_key(
+                key_file.read(),
+            )
+
     # Create video folder (if necessary)
     if not os.path.exists(app.config["VIDEO_FOLDER"]):
         os.mkdir(app.config["VIDEO_FOLDER"])
 
 
+def encrypt(message):
+    """Encrypt message with public key"""
+
+    encrypted_message = app.config["PUBLIC_KEY"].encrypt(
+        message.encode(),
+        padding.OAEP(
+            mgf=padding.MGF1(algorithm=hashes.SHA256()),
+            algorithm=hashes.SHA256(),
+            label=None,
+        ),
```

Comment:
What do these arguments do exactly and why were they chosen? Please add a comment.